### ISSUE TITLE
OpenAPI extension fields(Extension and ExtensionProperty) not working as expected with micronaut-openapi

### ISSUE THREAD

**Issue by leenanimbekar** [micronaut-projects/micronaut-openapi#28]
In a Controller and Application.Kt file I am trying to add additional fields i need using Extension annotation from OpenApi but the format is wrong in open api.yml file.
```
 @Get("/xxx")
 @ApiResponses(value = arrayOf(
            ApiResponse(responseCode = "200", description = "Success"),
            ApiResponse(responseCode = "400", description = "Bad Request"),
            ApiResponse(responseCode = "405", description = "Method Not Allowed")
    ))
@Operation(extensions = arrayOf(
           Extension(properties = arrayOf(
                ExtensionProperty(name = "x-abc", value = "xxx")
    ))))
public Dummy dummyAnnot() {
        return new Dummy();
    }
```

**Expected Output -**
```
openapi: 3.0.1
info:
  title: xxx
  description: RESTful API for xxx.
  contact:
    name: xxx
    email: xxx@xx.com
  version: V1
paths:
  /xxx:
    get:
      operationId: xxx
      parameters: []
      x-abc: xxx
      responses:
        200:
          description: Success
        400:
          description: Bad Request
        405:
          description: Method Not Allowed
```
**Actual Output -**
```      
openapi: 3.0.1
info:
  title: xxx
  description: RESTful API for xxx.
  contact:
    name: xxx
    email: xxx@xx.com
  version: V1
paths:
  /xxx:
    get:
      operationId: xxx
      parameters: []
      properties: 
         name: x-abc
         value: xxx
      responses:
        200:
          description: Success
        400:
          description: Bad Request
        405:
          description: Method Not Allowed 
```

the additional fields are getting added as 
```
properties: 
         name: x-abc
         value: xxx
``` 
whereas expected would be 
```
x-abc: xxx
```

**Comment by leenanimbekar** [comment]
Can someone please help here ?????

**Comment by graemerocher** [comment]
Currently extensions properties are not supported. I have marked this as an improvement. It has not gone unnoticed and we will get to it as soon as someone becomes available to work on it.

If you wish you to resolve the issue sooner this is an Open Source project and you can send a PR. The `Operation` annotation is parsed here https://github.com/micronaut-projects/micronaut-openapi/blob/master/openapi/src/main/java/io/micronaut/openapi/visitor/OpenApiControllerVisitor.java#L132

So that is where to begin to look. 

Test cases can be found here https://github.com/micronaut-projects/micronaut-openapi/blob/master/openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiOperationParseSpec.groovy

**Comment by leenanimbekar** [comment]
Thanks for the update !

**Comment by puneetbehl** [comment]
I think it is fixed with #73
